packages/block-library/src/template-part/editor.scss

```diff
@@ -77,6 +77,34 @@
 	}
 }
 
+.wp-block-template-part__label-container {
+	position: absolute;
+	bottom: 100%;
+	width: 100%;
+
+	.wp-block-template-part__label-layout {
+		display: flex;
+		justify-content: flex-start;
+	}
+
+	.wp-block-template-part__label-content {
+		display: flex;
+		align-items: center;
+		border: 1px solid $gray-600;
+		border-bottom: none;
+		border-top-right-radius: $radius-block-ui;
+		border-top-left-radius: $radius-block-ui;
+		padding: 0 6px;
+	}
+
+	.wp-block-template-part__label-text {
+		color: $black;
```

Comment:
Im not sure if we want to define the color here.  It will be already inherited from a higher level (which currently is black but may update in other circumstances like use of a dark theme? 🤔 )